src/main/java/com/SpringBootApplication/springapplication.java

```diff
@@ -0,0 +1,21 @@
+package com.SpringBootApplication;
```

Comment:
package name should be in snake_case.

Reply:
Not completed.Due to some issues iam  unable to do this.

Reply:
completed 160fa0ee8e765ce06c6b9b62287e93427f5cb3dc